packages/ui/src/components/DataStory/clients/WorkspacesApi.ts

```diff
@@ -0,0 +1,14 @@
+export interface Tree {
+  path: string;
+  type: 'file' | 'folder';
+  content?: string;
+  children?: Tree[];
+}
+
+export interface WorkspacesApi {
+  get: ({ path }) => Promise<Tree>
```

Comment:
Does the `get` command recursively list files of all depths?

Reply:
For now, yes. It might be a problem for large trees but we can ignore that for now.